### PR TITLE
feat(media): add onFirstFrame and onAspectRatioChange

### PR DESCRIPTION
**Pull request by drymarau**
This should help build UIs that react to changes in aspect ratio

### CODE REVIEW

File: sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt [hunk 1]
```diff
@@ -139,10 +142,12 @@ fun ConferenceCallScreen(
                     modifier = Modifier
                         .align(Alignment.TopEnd)
                         .fillMaxWidth(0.25f)
-                        .aspectRatio(aspectRatio),
+                        .then(aspectRatioModifier),
```
Comment (tpettersen) [inline sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt:145]: For selfView I am seeing som odd things on tablet emulator (pixel C - API 33) - which I cannot reproduce on my phone. Are you able to repro?

When doing orientation changes, I get wrong aspect ratios for selfView in portrait (and even wrong orientatition in normal portrait - but correct in reverse portrait) and upside-down in one landscape: 
image
image

image

Comment (drymarau) [reply sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt:145]: Seems like a problem in the emulator, this PR doesn't rotate the frames (which is pretty evident at least in some of these screenshots).

Comment (tpettersen) [reply sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt:145]: But it does do aspect ratio, which is failing in the emulator for some reason. 🤔 
I will do a bit more testing then

Comment (drymarau) [reply sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt:145]: It just means that emulator is capturing in landscape when it's in portrait.

**Review by tpettersen — APPROVED** [review]
Works on my phone :)

**Pull request merged**